src/index.js

```diff
@@ -1,20 +1,23 @@
-import { StrictMode } from 'react'
-import { render } from 'react-dom'
-import { BrowserRouter } from 'react-router-dom'
-import GlobalThemeProvider from './themes/GlobalThemeProvider'
-import { EmailContextProvider } from './custom/EmailContext'
-import App from './App'
-import './assets/style.scss'
-
-render(
-	<StrictMode>
-		<BrowserRouter>
-			<GlobalThemeProvider>
-				<EmailContextProvider>
-					<App />
-				</EmailContextProvider>
-			</GlobalThemeProvider>
-		</BrowserRouter>
-	</StrictMode>,
-	document.getElementById('root')
-)
+import { StrictMode } from 'react'
+import { render } from 'react-dom'
+import { BrowserRouter } from 'react-router-dom'
+import GlobalThemeProvider from './themes/GlobalThemeProvider'
+import { EmailContextProvider } from './custom/EmailContext'
+import { FirebaseContextProvider } from './custom/_firebaseContext'
+import App from './App'
+import './assets/style.scss'
+
+render(
+	<StrictMode>

```

Comment:
haha, we are definitely hitting that GlobalContext Hell. I am not even sure that is a thing 😅